docs/user/tutorials/index.rst

```diff
@@ -0,0 +1,41 @@
+.. this page is referenced from the front page but it's unnecessary as a navigation section for now.
+
+:orphan:
+
+=========
+Tutorials
+=========
+
+⏩️ :doc:`/tutorial/index`
+  This is where you should go if you are trying Read the Docs for the first time!
+
+
+⏩️ :doc:`/intro/getting-started-with-sphinx`
+  Sphinx is the most popular documentation tool on our platform.
+  But that's not to say that it's *easy* if you are using it for the first time.
+  We recommend a soft start with this tutorial.
```

Comment:
Truthy narrative here, but I'd be left with more questions after reading this. This explanation might be better for that guide, and with some more details. Or perhaps just a "Choosing the right tool" web page, documentation, or blog post.

However, I do really like the commentary here on each tool: "We suggest this tool... " is a fantastic start to maybe a larger document comparing tools.

Not a firm suggestion, but I'd stick with the positives of each tool and keep up with the suggestions for tool fit:

```suggestion
  It supports both reStructuredText and Markdown formats, and can generate rich API documentation from your source code.
  We recommend Sphinx for documentation that includes API reference documentation.
```